### PR TITLE
OCPBUGS-13925 add flags to collect podman logs

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s): 4.12.z

Issue:
https://issues.redhat.com/browse/OCPBUGS-13925

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
/lgtm, I wonder if this module though should be renamed `support-generating-a-sos-report-archive.adoc` for clarity. Might create more work than it is worth though.

**Comment by jmanthei** [comment]
@sunilcio QE plz

**Comment by kalexand-rh** [comment]
@sunilcio, will you please review this PR?

**Comment by mburke5678** [comment]
@jmanthei Can you confirm which versions this PR should be cherrypicked to? 4.10+?

**Comment by jmanthei** [comment]
@mburke5678 4.12 only I believe

**Comment by mburke5678** [comment]
@sunilcio Can you verify which versions this change applies to? I see that the original command goes back to 4.10+

**Comment by jmanthei** [comment]
/label merge-review-needed

Had to rebase but should be good to go now!

**Comment by kcarmichael08** [comment]
Hi Jessi, can you squash commits please?

**Comment by kcarmichael08** [comment]
Removing merge-review-in-progress label, because I think it was left there by mistake. After squashing, should be ready for merge review and leaving the merge-review-needed label on will ensure that it gets reviewed. I'm about to sign out so another merge reviewer can pick it up.

File: modules/support-generating-a-sosreport-archive.adoc [hunk 1]
```diff
@@ -81,7 +81,7 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 +
 [source,terminal]
 ----
-# sosreport -k crio.all=on -k crio.logs=on <1>
+# sos report -k crio.all=on -k crio.logs=on  -k podman.all=on -k podman.logs=on <1>
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply modules/support-generating-a-sosreport-archive.adoc:89]: Are we sure this should be `sosreport` https://github.com/openshift/openshift-docs/pull/65978/files#r1354852484

[the remark this answers is not in the document]
Comment (jmanthei) [reply modules/support-generating-a-sosreport-archive.adoc:89]: @sunilcio does this only apply to 4.12 or ...?

[the remark this answers is not in the document]
Comment (sunilcio) [reply modules/support-generating-a-sosreport-archive.adoc:89]: @JoeAldinger @jmanthei I checked and see both `sos report` and `sosreport` works. Checking previous comment I see `sosreport` is deprecated already with latest toolbox container. So `sos report` is correct

**Pull request merged**